docs/ldml/tr35-keyboards.md

```diff
@@ -485,10 +488,10 @@ Element containing informative properties about the layout, for displaying in us
 **Syntax**
 
 ```xml
-<info [author="{author}"]
-      [normalization="{form}"]
-      [layout="{hint of the layout}"]
-      [indicator="{short identifier}"] />
+<info author="{author}"
+      normalization="{form}"
+      layout="{hint of the layout}"
+      indicator="{short identifier}" />
```

Comment:
Why did you drop the `[` `]` which indicated optionality? Having that visible was really helpful I thought.

Reply:
it was inconsistent with other parts of the spec, and also led to the XML being not quite XML.  

Perhaps something like: `author="{author (optional)}"`

Reply:
I guess. The square brackets were really clear I thought but consistency probably trumps that :grin:

Reply:
I also prefer the square brackets and we possibly introduced them as I was fixing the keyboard LDML earlier. They should make the other parts of the spec match this one! :-)